### ISSUE TITLE
lsd can't be enabled for groups

### ISSUE THREAD

**Issue by harcesz** [jancborchardt/lsd#5]
I get a "lsd can't be enabled for groups" when trying to enable only for a single group. Sadly we still have some sane people working in our office, and they might prefer a less trippy working env.

**Comment by jancborchardt** [comment]
That's strange, I have no idea why that is. @BernhardPosselt @MorrisJobke does this require anything special in the app? (We might need to adjust the app tutorial then.)

**Comment by MorrisJobke** [comment]
Yes - setting the type not to "authentication", because authentication apps can't be enabled for groups only. Also no non-authentication app should have this, because those apps are loaded differently.

https://github.com/jancborchardt/lsd/blob/master/appinfo/info.xml#L12

**Comment by jancborchardt** [comment]
@MorrisJobke ok – I was under the impression that the type is necessary for an app which runs on the log in screen? Cause we set that for the Unsplash app too: https://github.com/jancborchardt/unsplash/blob/master/appinfo/info.xml

Then separately though, for auth apps, the "enable only for groups" interface should not be shown in the apps management. Or where did you try to set this and encountered the error @harcesz?

**Comment by MorrisJobke** [comment]
Usually this should already be the case, otherwise it is an regression.

The type only specifies when an app is loaded in the request lifecycle and not where it could integrate. Could you open a doc issue for this?

**Comment by harcesz** [comment]
Cant find it atm, I think that might have been directly in the apps installation/management menu, but can't find it now :/
